com.unity.netcode.gameobjects/Runtime/SceneManagement/SceneEventProgress.cs

```diff
@@ -84,7 +88,7 @@ internal class SceneEventProgress
         /// </summary>
         internal bool AreAllClientsDoneLoading { get; private set; }
 
-        internal string SceneName { get; set; }
+        internal uint SceneIndex { get; set; }
```

Comment:
Scene[What]Index ?

Reply:
Well, it is short for "SceneBuildIndex".  I will change that in another PR I have open.